taca/server_status/cronjobs.py

```diff
@@ -9,35 +9,42 @@
 
 from taca.utils.config import CONFIG
 
-def update_cronjob_db():
-    logging.info('Getting list of cronjobs')
-    crontab = CronTab()
-    # CronTab() is supposed to return the list of cronjobs
-    # it works on milou, but doesn't work on preproc (returns an empty list)
-    # when it doesn't work, create CronTab object from the output of command 'crontab -l'
-    if crontab.crons == []:
+def _parse_crontab():
+    result = {}
+    user = getpass.getuser()
+    logging.info('Getting crontab for user {}'.format(user))
+    crontab = None
+    try:
+        crontab = CronTab(user=user)
+    except Exception, e:
+        logging.warning('Cannot get a crontab for user: {}'.format(user))
+        logging.warning(e.message)
         try:
-            p = subprocess.Popen('crontab -l', shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+            logging.info('Getting a crontab for default user')
+            crontab = CronTab()
```

Comment:
No, this one returns crontab for current user. 
This was the previous implementation, but for some reasons it doesn't parse comments